docs/tutorials/control/scuttle-gamepad.md

```diff
@@ -87,16 +112,38 @@ Copy and paste the following into the empty **Attributes** field, replacing `THE
 
 ![Service configuration](/tutorials/scuttle-gamepad/gamepad-service-config.png)
 
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
+
+If instead of using the config builder, you prefer to write raw JSON, switch to **Raw JSON** mode on the **Config** tab.
+Add the following configuration for your base remote control service, replacing `<your-base-name>` with your base's name:
```

Comment:
```suggestion
If instead of using the config builder, you prefer to write raw JSON, switch to [**Raw JSON** mode](/manage/configuration/#the-config-tab) on the **Config** tab.
Inside the `services` array of your config, add the following configuration for your base remote control service, replacing `<your-base-name>` with your base's name:
```

Reply:
```suggestion
If instead of using the config builder, you prefer to write raw JSON, switch to [**Raw JSON** mode](/manage/configuration/#the-config-tab) on the **Config** tab.
Add the following configuration for your base remote control service, replacing `<your-base-name>` with your base's name:
```
This one is different from the components part, because a new robot in the Viam app autopopulates with just a components array and no services array so unless they've already been doing other things with services on this robot, they don't have a services array to add to yet; they need to make it.